OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2399,6 +2399,32 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				// Notify how to restore blocking movement of Mobile actors with Carryable and/or Cargo
+				if (engineVersion < 20151121 && depth == 2)
+				{
+					if (node.Key == "Carryable")
+					{
+						Console.WriteLine("Use CarryableUpgrades of Carryable to disable Mobile while " + parent.Key + " is waiting or being carried.");
+					}
+					else if (node.Key == "Cargo")
+					{
+						Console.WriteLine("Use LoadingUpgrades of Cargo to disable Mobile while " + parent.Key + " is loading cargo.");
+					}
+					else if (node.Key == "DeployToUpgrade")
+					{
+						Console.WriteLine("Use Upgrades of DeployToUpgrade to disable Mobile while " + parent.Key + " is deployed (instead of DisableUpgrade).");
+					}
+					else if (node.Key == "DisableUpgrade")
+					{
+						Console.WriteLine("DisableUpgrade no longer disables Mobile. Use Mobile upgradablility instead for " + parent.Key + ".");
```

Comment:
Maybe only print those messages when one of the sibling nodes contains `DisableMovementOnUpgrade`? Otherwise running the upgrade rules would be quite spammy.